### DISCUSSION TITLE
Student Developer Pack: can't re-apply (trying it since March 16)

### DISCUSSION THREAD

**Discussion by dabzse** [community/community#120057] [GitHub Education]
### Hi everyone,

my Student Developer Pack has been expired in February, but I didn't need it then. we started our new semester in March, so I tried to re-apply then.
I've created a GiitHub Support ticket, but their answer was an auto-reply; the tickets are archived, so I can't reply to them. and also can't create a new ticket or follow-up ticket, because it is no longer supported. their answers are not really useful.

in 2022 this University was a good fit, so should be a good fit nowadays too.
and also, I have included my student ID card's both sides: 1024x768 and 100kB < size < 1MB

what am I missing? or what did Microsoft crashed?
(because not I am the only who faces these problems)

**Comment by samus-aran** [comment]
Hi @dabzse 👋 

Can you explain in more detail what the issues you are facing that is stopping you from re-applying?

**Reply by dabzse** [reply]
Upload proof of your academic status
Already have an image? We accept JPEGs and PNGs of at least 1024x768 resolution between 100KB and 1MB in size.

student ID card, used in Hungary (not the same as in the USA);
1024x768 PNG about 800kB in size.
scanned with a scanner and modified with gimp to fit the size. originally scanned size: 1014x636.
if I upload the front or a back, I can't go forward. 2 years ago there was no problem after uploading both sides (without the currently required minimal resolution)
